test/services/courses/services/coursePermission.test.js

```diff
@@ -7,39 +7,15 @@ const testObjects = require('../../helpers/testObjects')(app);
 const coursePermissionService = app.service('/courses/:scopeId/userPermissions');
 
 describe('PermissionService', async () => {
-	const userPermissions = [
-		'BASE_VIEW',
-		'DASHBOARD_VIEW',
+	const studentPermissions = [
```

Comment:
@CordlessWool and I had a discussion about this. We believe they are, so that you cannot change them without breaking at least one test. Let's discuss this again; I don't think it's pretty :-)